rotkehlchen/data/importer.py

```diff
@@ -1435,3 +1446,87 @@ def import_bisq_trades_csv(self, filepath: Path, **kwargs: Any) -> Tuple[bool, s
                 except KeyError as e:
                     return False, str(e)
         return True, ''
+
+    @staticmethod
+    def _group_binance_rows(
+        rows: List[BinanceCsvRow],
+        **kwargs: Dict[str, Any],
+    ) -> Tuple[int, Dict[Timestamp, List[BinanceCsvRow]]]:
+        multirows: Dict[Timestamp, List[BinanceCsvRow]] = defaultdict(lambda: [])
+        skipped_count = 0
+        formatstr = kwargs.get('timestamp_format')
+        for csv_row in rows:
+            try:
+                timestamp = Timestamp(deserialize_timestamp_from_date(
+                    date=csv_row['UTC_Time'],
+                    formatstr=formatstr if formatstr is not None else '%Y-%m-%d %H:%M:%S',  # type: ignore  # noqa: E501
+                    location='binance',
+                ))
+                csv_row['Coin'] = asset_from_binance(csv_row['Coin'])
+                csv_row['Change'] = deserialize_asset_amount(csv_row['Change'])
+                del csv_row['UTC_Time']
+                del csv_row['User_ID']
+                del csv_row['Account']
+                del csv_row['Remark']
+                multirows[timestamp].append(csv_row)
+            except (DeserializationError, UnknownAsset) as e:
+                log.warning(f'Skipped binance csv row {csv_row} because of {str(e)}')
+                skipped_count += 1
+            except KeyError as e:
+                log.error(f'Malformed binance csv columns! Broke on row {csv_row}. {str(e)}')
+                return len(rows), {}
+
+        return skipped_count, multirows
+
+    def _process_binance_rows(self, multi: Dict[Timestamp, List[BinanceCsvRow]]) -> None:
+        BinanceEntry.db = self.db
+        single_entry_classes = BinanceSingleEntry.__subclasses__()
+        multiple_entry_classes = BinanceMultipleEntry.__subclasses__()
+        stats: Dict[Type[BinanceEntry], int] = defaultdict(lambda: 0)
```

Comment:
`defaultdict(fun)` calls `fun()` and the default value of `int()` is `0`

```suggestion
        stats: Dict[Type[BinanceEntry], int] = defaultdict(int)
```

Reply:
Cool, didn't know it, thanks!